src/core/task10.js

```diff
@@ -1,13 +1,20 @@
-export var obj = {
-      first: prompt("Введите первое число"),
+export default function Number() {
```

Comment:
It's a very bad practice to override reserved words in every programming language. 
Rewrite it using another word.